Whiskey/Tasks/Set-WhiskeyVersion.ps1

```diff
@@ -0,0 +1,359 @@
+
+ function Set-WhiskeyVersion 
+{
+    <#
+    .SYNOPSIS
+    Sets the version for the current build.
+
+    .DESCRIPTION
+    The `Version` task sets the version for the current build. Whiskey only supports [semantic versions](http://semver.org).
+    
+    You can set the version explicitly with the `Version` property. Whiskey can read the version from a .NET Core .csproj file, a PowerShell module manifest, or a Node package.json file. Set the `Path` property to the path to the file to read from. If it is unsupported or doesn't contain a version, you'll get an error.

```

Comment:
This sentence says if the version cannot be found from a file given with the `Path` property then an error will be given when in reality the entire build will fail.